app/src/main/AndroidManifest.xml

```diff
@@ -37,6 +37,11 @@
                 android:name="android.support.FILE_PROVIDER_PATHS"
                 android:resource="@xml/file_paths" />
         </provider>
+
+        <activity
+            android:name=".Leaderboard"
+            android:label="@string/title_activity_leaderboard"
+            android:theme="@style/AppTheme"></activity>
```

Comment:
Since the Leaderboard is not an activity you can remove this part from the manifest.

Reply:
Thank you, auto-generated stuff, I didn't see it